int32 pressure
geometry_msgs/Vector3 gyro
geometry_msgs/Vector3 acc
geometry_msgs/Vector3 mag
uint16 temp_gyro
uint32 temp_ADC